### DISCUSSION TITLE
Add sub-context menu items

### DISCUSSION THREAD

**Discussion by danijelVR** [Krypton-Suite/Standard-Toolkit#1884] [Q&A] [answered]
Hello,
Is it possible to add sub-menu items on the existing context menu item, as it is shown on the image bellow?
So when I press "Print", two more items should appear on the right.
![063m0Qrwoq](https://github.com/user-attachments/assets/2d97de49-6de5-421f-86a3-0819521f6d1d)

Any help will be very appreciated.

**Comment by Smurf-IV** [comment, marked as answer]
Answered in #1885 1885
